.github/workflows/merge-queue.yml

```diff
@@ -120,6 +120,24 @@ jobs:
 
       - name: "Python: TensorZero Client: pytest"
         working-directory: clients/python
+        run: |
+          source .venv/bin/activate
+          ./test.sh
+
```

Comment:
@Aaron1011 I think this got trampled and should be fixed (test.sh for the new client and pytest for old)